go.mod

```diff
@@ -10,6 +10,8 @@ replace cuelang.org/go => github.com/grafana/cue v0.0.0-20230926092038-971951014
 // import that instead of v0.X even though v0.X is newer.
 replace github.com/prometheus/prometheus => github.com/prometheus/prometheus v0.52.0
 
+replace github.com/grafana/alerting => /Users/titolins/dev/alerting
```

Comment:
Leaving this here just as a reminder to update alerting once https://github.com/grafana/alerting/pull/247 has been merged